src/reducers/app/index.js

```diff
@@ -81,7 +82,7 @@ export default (state = {
       return Object.assign({}, state, {
         vegaSpec: spec,
         mode: MODES.Vega,
-        editorString: action.spec,
+        editorString: JSON3.stringify(spec, null, 2, 60),
```

Comment:
I don't like this because this means we always format the spec. We actually hand format the specs for vega-lite.